### PR TITLE
Add PCQ to aat

### PR DESCRIPTION
**Pull request by bradley-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PCQ-109

### Change description ###

Add PCQ config to aat environment.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -222,3 +222,7 @@ k8s/perftest/common/xui/webapp.yaml @hmcts/xui
 k8s/perftest/common/xui/manage-organisations.yaml @hmcts/xui
 k8s/perftest/common/xui/approve-org.yaml @hmcts/xui
 k8s/perftest/common/xui/share-case.yaml @hmcts/xui
+
+### PCQ
+k8s/**/common/pcq/namespace.yaml @hmcts/pcq-admins
```
Comment (adusumillipraveen) [inline CODEOWNERS]:
```suggestion
k8s/aat/common/pcq/namespace.yaml @hmcts/pcq-admins
```

File: CODEOWNERS [hunk 2]
```diff
@@ -222,3 +222,7 @@ k8s/perftest/common/xui/webapp.yaml @hmcts/xui
...
+k8s/**/common/pcq/identity.yaml @hmcts/pcq-admins
```
Comment (adusumillipraveen) [inline CODEOWNERS]:
```suggestion
k8s/aat/common/pcq/identity.yaml @hmcts/pcq-admins
```

File: CODEOWNERS (hunk 1 above)
Comment (bradley-hmcts) [reply CODEOWNERS]: Resolved in latest commmit

File: CODEOWNERS (hunk 2 above)
Comment (bradley-hmcts) [reply CODEOWNERS]: Resolved in latest commmit

**Pull request merged**